kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/CaseManagementContainmentControl.java

```diff
@@ -137,12 +137,14 @@ public Point2D getAdjust() {
 
     @Override
     public void execute() {
-        // Execute.
-        final CaseManagementShapeView ghost = state.getGhost().get();
-        //Children contains m_ghost and others excluding m_shape. This replaces m_ghost with m_shape.
-        final WiresContainer originalParent = state.getOriginalParent().get();
-        reparentDraggedShape(ghost,
-                             originalParent);
+        if (state.getGhost().isPresent() && state.getOriginalParent().isPresent()) {
```

Comment:
just a suggestion on this block:
```suggestion
state.getGhost()
    .ifPresent(ghost -> state.getOriginalParent()
         .ifPresent(originalParent -> reparentDraggedShape(ghost, originalParent)));
```